packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -1014,40 +1057,59 @@ export class ReadAlongComponent {
-    if (props.attributes && props.attributes['xml:lang']) {
+    if (attributes && attributes['xml:lang']) {
 
-      nodeProps['lang'] = props.attributes['xml:lang'].value
+      nodeProps['lang'] = attributes['xml:lang'].value
     }
-    if (props.attributes && props.attributes['lang']) {
+    if (attributes && attributes['lang']) {
 
-      nodeProps['lang'] = props.attributes['lang'].value
+      nodeProps['lang'] = attributes['lang'].value
     }
 
     return <div {...nodeProps}
-      class={'sentence' + " " + (props.attributes["class"] ? props.attributes["class"].value : "")}>
+      class={'sentence' + " " + (attributes["class"] ? attributes["class"].value : "")}>
       {
         /* Here are the Word and NonWordText children */
-        props.words.map((child: Element, c) => {
+        words.map((child: Element, c) => {
 
           if (child.nodeName === '#text') {
             return <this.NonWordText text={child.textContent} attributes={child.attributes}
-              id={(props.attributes["id"] ? props.attributes["id"].value : "P") + 'text' + c} />
+              id={(attributes["id"] ? attributes["id"].value : "P") + 'text' + c} />
           } else if (child.nodeName === 'w') {
             return <this.Word text={child.textContent} id={child['id']} attributes={child.attributes} />
           } else if (child) {
             let cnodeProps = {};
-            if (child.attributes['xml:lang']) cnodeProps['lang'] = props.attributes['xml:lang'].value
-            if (child.attributes['lang']) cnodeProps['lang'] = props.attributes['lang'].value
+            if (child.attributes['xml:lang']) cnodeProps['lang'] = attributes['xml:lang'].value
+            if (child.attributes['lang']) cnodeProps['lang'] = attributes['lang'].value
             return <span {...cnodeProps} class={'sentence__text theme--' + this.theme + (' ' + child.className)}
               id={child.id ? child.id : 'text_' + c}>{child.textContent}</span>
           }
         })
       }
+      {(() => {
+       if (this.mode === 'EDIT') {
+          if (sentenceID in this.translations && sentenceID in this.translations && this.translations[sentenceID] !== null) {
+            return <span class="sentence__translation">
+              <button data-cy="remove-translation-button" onClick={() => this.removeLine(props.sentenceData)}>{this.returnTranslation('remove-line', this.language)}</button>
+              <p data-cy="translation-line" class="sentence__text" onInput={(e: any) => {this.updateTranslation(sentenceID, e.currentTarget.innerText)}} contentEditable data-placeholder={this.returnTranslation('line-placeholder', this.language)}></p>
+              </span>
+          } else {
+            return <button data-cy="add-translation-button" class="sentence__translation" onClick={() => this.addLine(props.sentenceData)}>{this.returnTranslation('add-line', this.language)}</button>
+          }
+        }
+       else {
+          return null
+       }
+   })()
+   }
```

Comment:
These lines are not indented correctly, and that made it harder to understand the code. We really need to enable prettier on this repo!